QUEST_LV_0100.tsv

```diff
@@ -9579,14 +9579,14 @@ QUEST_LV_0100_20151001_009574	{memo X}You have to save the mysterious girl while
 QUEST_LV_0100_20151001_009575	{memo X}You've obtained the stone!
 QUEST_LV_0100_20151001_009576	You couldn't obtain anything!
 QUEST_LV_0100_20151016_009577	
-QUEST_LV_0100_20151016_009578	
-QUEST_LV_0100_20151016_009579	
+QUEST_LV_0100_20151016_009578	$You mean you want to enter the Crystal Mines to find the evidence of Salvation? {nl}
+QUEST_LV_0100_20151016_009579	$So we are not in the situation to spare our troops for guarding Revelators while they check the evidence of salvation. {nl}But we also can't go against the order of the Bishop and Knight Commander either. {nl}
```

Comment:
We should leave it as 'evidence of salvation' - Checked ITEM real quick, there's only "Relic of Salvation" used in the Aurelius story.